gobblin-runtime/src/main/java/gobblin/runtime/JobListener.java

```diff
@@ -19,7 +19,14 @@
   /**
    * Called when a job is completed.
    *
-   * @param jobState Job state
+   * @param jobState a {@link JobState} object
    */
-  public void jobCompleted(JobState jobState);
+  public void onJobCompletion(JobState jobState);
```

Comment:
JobListenter is a public API. We have to get into the habit of not changing public APIs in a backwards-incompatible manner unless absolutely necessary.